licensing/src/main/java/io/crate/license/Cryptos.java

```diff
@@ -62,4 +96,11 @@ private Cryptos() {
             throw new IllegalStateException(e);
         }
     }
+
+    static byte[] digest(byte[] inputBytes) {
+        MessageDigest sha256 = MessageDigests.sha256();
```

Comment:
We don't need the thread-local facility of `MessageDigests` afaik - so we could go with plain `MessageDigest.getInstance(digest)`. I wonder if we should expose the used hash function to the API user? E.g. `sha256Digest(..)` ?